libs/framework/include/celix/TrackerBuilders.h

```diff
@@ -42,9 +42,9 @@ namespace celix {
         //NOTE private to prevent move so that a build() call cannot be forgotten
         ServiceTrackerBuilder(ServiceTrackerBuilder&&) noexcept = default;
     public:
-        explicit ServiceTrackerBuilder(std::shared_ptr<celix_bundle_context_t> _cCtx, std::string_view _name) :
+        explicit ServiceTrackerBuilder(std::shared_ptr<celix_bundle_context_t> _cCtx, std::string _name) :
```

Comment:
I think `ServiceTrackerBuilder(std::shared_ptr<celix_bundle_context_t> _cCtx, std::string&& _name)`, which eliminator a constructor call, is better.

And the best solution I know is so called universal reference, from Effective Modern C++, Item 25:

```C++
template<typename T>
explicit ServiceTrackerBuilder(std::shared_ptr<celix_bundle_context_t> _cCtx, T&& _name) :
cCtx{std::move(_cCtx)},
name{std::forward<T>(_name)} {}

```

If `_name` is a plain C string, this will also work without extra constructor calls. 
This consideration should apply to several other places. 

I also wonder if usage of universal reference will make string_view unnecessary? See `Trackers.h` for an example.

Reply:
Taking Item 41 into account, passing `_name` by value is indeed a good choice. Resolved.